performance/language_normalization.rb

```diff
@@ -0,0 +1,480 @@
+# frozen_string_literal: true
```

Comment:
Agreed! I believe the directory used to be named `scripts` but was just renamed within the past week to `performance`. Anyone have a better idea where it belongs?

Reply:
I'm not sure where it fits. It almost feels like maybe we don't want to check this in at all, since it is in theory a one time thing (that being said it's probably easiest for us to review if you leave it for now).